src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -169,3 +184,12 @@ def get_fuzz_target_weights():
     weights[fuzz_target.binary] = target_job.weight
 
   return weights
+
+
+def get_job_list(jobs_str):
+  if jobs_str:
+    jobs_list = [job.strip() for job in jobs_str.split(",")] if jobs_str else []
```

Comment:
I don't think this ternary check is necessary since you already verified that `jobs_str` exists in the `if` statement. It might just be cleaner to do 

```
return [job.strip() for job in jobs_str.split(",")]
```

Reply:
Done